src/services/api.service.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+export const baseAxios = axios.create({ baseURL: 'https://library-practice-app.azurewebsites.net/api' })
+baseAxios.interceptors.request.use(
+  config => {
+    const token = localStorage.getItem('token')
```

Comment:
You can get token from service here as well

Reply:
Oops, I missed that case